spec/integration/util/windows/security_spec.rb

```diff
@@ -368,7 +368,7 @@ def grant_everyone_full_access(path)
         end
       end
 
-      describe "for an administrator", :if => Puppet.features.root? do
+      describe "for an administrator", :if => (Puppet.features.root? && Puppet.features.microsoft_windows?) do
```

Comment:
I know, it seems completely unnecessary, right? But rspec doesn't AND nested include/exclude filters, it ORs them. So the following `inner` example group will be executed even though the `outer `one is excluded.

<pre>
describe "outer", :if => false do
  describe "inner", :if => true do
    it "executes" do
      puts "hello"
    end
  end
end
</pre>

which when executed:

<pre>
$ bundle exec rspec spec/unit/crazy_spec.rb
Hello
.

Finished in 0.00035 seconds
1 example, 0 failures
</pre>